doc/manual/src/language/import-from-derivation.md

```diff
@@ -0,0 +1,94 @@
+# Import From Derivation
```

Comment:
```suggestion
# Reading build results
```

Reply:
Let us do the renaming in another pull request to focus on the essence of this one. See my [top-level comment](https://github.com/NixOS/nix/pull/7332#issuecomment-1324800375).